products/memgraph/upcoming/concepts/indexing.md

```diff
@@ -17,9 +17,10 @@ At Memgraph, we support two types of indexes:
   * label index
   * label-property index
 
-Label indexing is enabled by default in Memgraph, i.e., Memgraph automatically
-indexes labeled data. By doing so we optimize queries which fetch nodes by
-label:
+Label indexing is not enabled by default in Memgraph, i.e., Memgraph will not
```

Comment:
I would emphasize the `NOT` part (caps)...

Reply:
Will do.